hack/make-rules/verify.sh

```diff
@@ -73,6 +73,17 @@ function is-quick {
   return 1
 }
 
+function is-explicitly-chosen {
+  local name="${1#verify-}"
+  name="${name%.*}"
+  for e in ${VERIFY_CHECKS}; do
+    if [[ $e == $name ]]; then
```

Comment:
nit: we generally use `${var}` form and quote variables.

Reply:
you don't need to quote variables in [[ blocks

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2053

I agree with @ixdy that we should use the `"${name}"` here

Reply:
Done.